pkgs/games/keeperrl/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchFromGitHub, SDL2, SDL2_image, openal, libvorbis, zlib, curl,
+  libGL, makeWrapper, boost }:
+
+stdenv.mkDerivation rec {
+  name = "keeperrl-${version}";
+  version = "alpha24";
+
+  src = fetchFromGitHub {
+    owner = "miki151";
+    repo = "keeperrl";
+    rev = "${version}";
+    sha256 = "00vpq4njdl8hqndjvx2r0w0azz9lfil4qg1rlhj3y9qvc1i0xwwn";
+  };
+
+  patches = [ ./makefile_cflags.patch ];
+
+  buildInputs = [
+    SDL2 SDL2_image openal libvorbis zlib curl libGL makeWrapper
+  ] ++ stdenv.lib.optional stdenv.isDarwin boost ;
+
+  makeFlags = [ "OPT=true" "RELEASE=true" ];
+
+  CFLAGS = "-I${SDL2.dev}/include/SDL2";
```

Comment:
What error is there when you remove this line?

Reply:
```
g++ -MMD -Wall -std=c++1y -Wno-sign-compare -Wno-unused-variable -Wno-shift-count-overflow -Wno-tautological-constant-out-of-range-compare -Wno-mismatched-tags -ftemplate-depth=512 -DDATA_DIR=\"ATA_DIR\" -DRELEASE -O3 -I. -I./extern  -c animation.cpp -o obj-opt/animation.o
In file included from sdl.h:11,
                 from renderer.h:19,
                 from animation.cpp:5:
/nix/store/9rgym24jflmh4lqw0xhv752xjhwai7yj-SDL2_image-2.0.5/include/SDL2/SDL_image.h:27:10: fatal error: SDL.h: No such file or directory
 #include "SDL.h"
```